src/components/auth/signup/Signup.css

```diff
@@ -0,0 +1,104 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.row {
+  display: flex;
+  height: 100%;
+  background-image: url('./SignupBg.jpg');
+  background-size: cover;
+}
+
+.leftDiv {
```

Comment:
Your use of `camelCasing` for CSS selectors is contradictory, style-wise, compared to how @codeshifu has already started off.